cogs/fun.py

```diff
@@ -69,21 +69,23 @@ def __init__(self, bot):
         await ctx.send(embed=embed)
 
     @commands.command()
-    async def delete(self, ctx, *, username: discord.Member):
+    async def delete(self, ctx, *, username):
         """Deletes the specified user."""
-        if username.id == 319503910895222784:
-            await ctx.send("You will not touch my boyfriend!")
-            return
-        elif username.id == 201745963394531328 or username.id == 206197394667208704:
-            await ctx.send("That's a funny joke, {}.".format(ctx.message.author.name))
-            return
-        elif username.id == 399315651338043392:
-            await ctx.send("You're so funny, {}.".format(ctx.message.author.name))
-            return
         try:
-            await ctx.send("``characters/{}.chr`` deleted successfully.".format(username.name.lower()))
+            username: discord.Member
```

Comment:
Is this correct Python syntax?

Reply:
no

Reply:
I just tested that one line. It works fine.